lahja/asyncio/endpoint.py

```diff
@@ -734,7 +747,9 @@ def broadcast_nowait(
 
         return result
 
-    def _remove_cancelled_future(self, id: str, future: "asyncio.Future[Any]") -> None:
+    def _remove_cancelled_future(
+        self, id: RequestID, future: "asyncio.Future[Any]"
+    ) -> None:
```

Comment:
nitpick: Is this Black? I think our convention has been to not add this final newline

Reply:
black